lib/lang/countries_lang_codes_table.dart

```diff
@@ -47,7 +48,7 @@ const _COUNTRIES_LANG_CODES_TABLE = {
   'ba': ['bs', 'hr', 'sr'],
   'bb': ['en'],
   'bd': ['bn'],
-  'be': ['nl', 'fr', 'de'],
+  CountryCode.BELGIUM: ['nl', 'fr', 'de'],
```

Comment:
`['nl', 'fr', 'de']` and 14 other lists are now being duplicated inside the `country_table.dart` file.
Maybe we should remove the duplication by doing
```
CountryCode.BELGIUM: CountryTable.table[CountryCode.BELGIUM].languages,
```
... here
?

Reply:
yes, that's awesome! to bad i missed that!

Reply:
ah, it doesn't work because languages isn't a const. Let me check that out tommorow!

Reply:
We can remove `const` here, I don't think it would hurt :)